modules/imaging_uploader/test/imaging_uploaderTest.php

```diff
@@ -25,93 +25,44 @@
  */
 class ImagingUploaderTestIntegrationTest extends LorisIntegrationTest
 {
-    /**
-     * Insert testing data
-     *
-     * @return void
-     */
-    function setUp()
-    {
-        parent::setUp();
-         $this->DB->insert(
-             "psc",
-             array(
-              'CenterID'  => '55',
-              'Name'      => 'TESTinPSC',
-              'Alias'     => 'tst',
-              'MRI_alias' => 'test',
-             )
-         );
-          $this->DB->insert(
-              "candidate",
-              array(
-               'CandID'    => '999999',
-               'CenterID'  => '55',
-               'UserID'    => '1',
-               'PSCID'     => '8889',
-               'ProjectID' => '7777',
-              )
-          );
-          $this->DB->insert(
-              "session",
-              array(
-               'ID'           => '111112',
-               'CandID'       => '999999',
-               'CenterID'     => '55',
-               'UserID'       => '1',
-               'MRIQCStatus'  => 'Pass',
-               'SubprojectID' => '6666',
-               'Visit'        => 'In Progress',
-               'Visit_label'  => 'TestVisitLabel',
-              )
-          );
-          $this->DB->insert(
-              "mri_upload",
-              array(
-               'UploadID'                 => '9999999',
-               'UploadedBy'               => 'test',
-               'UploadDate'               => '2017-01-01',
-               'UploadLocation'           => '/data/incoming/test',
-               'DecompressedLocation'     => '/tmp/ImagingUpload-test',
-               'InsertionComplete'        => '0',
-               'Inserting'                => '0',
-               'PatientName'              => 'TestTestTest',
-               'number_of_mincInserted'   => '1',
-               'number_of_mincCreated'    => '1',
-               'TarchiveID'               => '999999',
-               'SessionID'                => '111112',
-               'IsCandidateInfoValidated' => '1',
-               'IsTarchiveValidated'      => '0',
-               'IsPhantom'                => 'N',
-              )
-          );
-    }
-    /**
-     * Deleting the test data
-     *
-     * @return void
-     */
-    function tearDown()
-    {
-        parent::tearDown();
-        $this->DB->delete(
-            "mri_upload",
-            array('PatientName' => 'TestTestTest')
-        );
-        $this->DB->delete(
-            "session",
-            array('CandID' => '999999')
-        );
-        $this->DB->delete(
-            "candidate",
-            array('CandID' => '999999')
-        );
-        $this->DB->delete(
-            "psc",
-            array('CenterID' => '55')
-        );
-
-    }
+    // expect UIs for Browse Tab
+    private $_loadingBrowseUI = array(
```

Comment:
From what I am seeing in the `_testPageUIs()` function , this array should contain Objects of the shape `array("label" => "the label to test against", "selector"=> "selector value")` and then use explicitly `$item["label"] and $item["selector"] in the forEach loop.

Reply:
Done.